qaframework-bdd-tests/cypress.json

```diff
@@ -6,7 +6,7 @@
   ],
   "video": true,
   "defaultCommandTimeout": 120000,
-  "baseUrl": "https://openmrs-spa.org/openmrs/spa",
+  "baseUrl": "https://dev3.openmrs.org/openmrs/spa",
   "env": {
     "API_BASE_URL": "https://openmrs-spa.org/openmrs/ws/rest/v1",
```

Comment:
Isn't the baseUrl supposed also to change?

Reply:
> Isn't the baseUrl supposed also to change?

Supposed to change to which url ?

Reply:
> Isn't the baseUrl supposed also to change?

Supposed to change to which url ?

Reply:
"API_BASE_URL": "https://dev3.openmrs.org/openmrs/ws/rest/v1",

Reply:
sorry i missed something here @kdaud @sherrif10 
just made the fix https://github.com/openmrs/openmrs-contrib-qaframework/pull/419